### PR TITLE
RM-7166 port imxrt1060 linux to up to date bsp baseline

### PR DESCRIPTION
**Pull request by asanochkin-emcraft**
Refer to https://sr.emcraft.com/rb/r/567/ for the unit test

### CODE REVIEW

File: drivers/video/fbdev/mxsfb.c [hunk 1]
```diff
@@ -39,7 +39,6 @@
  * of the i.MX28 and you can vary it between 2 ns ... 8 ns in 2 ns steps. Give
  * the required value in the imx_fb_videomode structure.
  */
-
```
Comment (vdskvortsov) [inline drivers/video/fbdev/mxsfb.c]: why this is needed?

Comment (vdskvortsov) [reply drivers/video/fbdev/mxsfb.c]: Why commit "RM-7166 port imxrt1060 linux to up to date bsp baseline" is needed?

**Comment by vdskvortsov** [comment]
Please add the formal file arch/arm/boot/dts/imxrt1060.dtsi, which can just include imxrt1050.dtsi for this moment. This imxrt1060.dtsi must be used in our projects for the IMXRT1060-EVK target.

**Review by vdskvortsov — CHANGES_REQUESTED** [review]

File: drivers/video/fbdev/mxsfb.c (hunk 1 above)
Comment (asanochkin-emcraft) [reply drivers/video/fbdev/mxsfb.c]: > why this is needed?
No needed. Will be fixed.

Comment (asanochkin-emcraft) [reply drivers/video/fbdev/mxsfb.c]: > RM-7166 port imxrt1060 linux to up to date bsp baseline
Sorry, but I do not see such commit in this PR. Could you clarify please?

Comment (vdskvortsov) [reply drivers/video/fbdev/mxsfb.c]: Sorry, this one: 

RM-7166: video: mxsfb driver: Remove the changes made in context of 5...

Comment (asanochkin-emcraft) [reply drivers/video/fbdev/mxsfb.c]: > Sorry, this one:
> 
> RM-7166: video: mxsfb driver: Remove the changes made in context of 5...
This is needed due to c958e7720c77c commit, I believe

Comment (vdskvortsov) [reply drivers/video/fbdev/mxsfb.c]: Understood. 
But please squash 73ee10b and 8913159 into a single commit, in order not to introduce changes that will be reversed by a subsequent commit.

File: drivers/video/fbdev/mxsfb.c [hunk 2]
```diff
@@ -1466,6 +1466,11 @@ static int mxsfb_init_fbinfo(struct mxsfb_info *host)
 
 	fb_info->fix.line_length =
 		fb_info->var.xres * (fb_info->var.bits_per_pixel >> 3);
+#if !defined(CONFIG_SOC_IMXRT)
```
Comment (vdskvortsov) [inline drivers/video/fbdev/mxsfb.c]: It seems the conflict was resolved incorrectly during cherry-picking. This change must be moved down, to line 1478, refer to the original commit 7283d34.

**Review by vdskvortsov — CHANGES_REQUESTED** [review]

File: drivers/video/fbdev/mxsfb.c (hunk 2 above)
Comment (vdskvortsov) [reply drivers/video/fbdev/mxsfb.c]: After fixing, remove the commits 73ee10b and 7ca78ba

Comment (asanochkin-emcraft) [reply drivers/video/fbdev/mxsfb.c]: Done

File: drivers/video/fbdev/mxsfb.c (hunk 1 above)
Comment (asanochkin-emcraft) [reply drivers/video/fbdev/mxsfb.c]: Done

**Review by vdskvortsov — APPROVED** [review]

**Pull request merged**